guake/data/org.guake.gschema.xml

```diff
@@ -86,6 +86,11 @@
             <summary>Tab on top.</summary>
             <description>Makes tab bar on top of the Guake window. Per default, tabs appears bellow the terminal. Setting this to True will move the tab on top of the terminal. Requires a restart.</description>
         </key>
+        <key name="new-tab-after" type="b">
```

Comment:
I would suggest that this option lives in the `Main Window Options` or `General` rather than in `Placement`, reasoning being that it's related to functionality of the tabs rather than general placement. Following the convention of `Set window title to current tab name` in `Main Window Options` or `Show close buttons for tabs` in `General`.

Reply:
Done.